packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -132,6 +132,15 @@ export interface AwsCustomResourceProps {
    */
   readonly policyStatements?: iam.PolicyStatement[];
 
+  /**
+   * The execution role for the Lambda function implementing this custom
+   * resource. The role must be assumable by the `lambda.amazonaws.com`
```

Comment:
The function is implementing the "custom resource *provider*". It's an important distinction because it's a singleton, so the role will apply to all `AwsCustomResource` instances in your app. Probably fine, but worth documenting.